README.md

```diff
@@ -119,13 +117,8 @@ After you're finished please remove all the comments and instructions!
 
 ## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [ Optional ] The `Getting Started` section of your readme file is very important, this means you have to fill it correctly. Please see an example of how I would fill in this section in my `readme` file. The instructions you give in the `readme` file should always be clear and easy for someone to follow.

<!-- GETTING STARTED -->

## 💻 Getting Started <a name="getting-started"></a>

To get a local copy up and running, follow these steps.

```sh
   git clone git@github.com:david30-maker/Mobile-first-project.git
```

### Prerequisites

In order to run this project you need:
 - NodeJS
 - Browser

### Setup

Clone this repository to your desired folder:

<!--
Example commands:
-->

```sh
  git clone git@github.com:david30-maker/Mobile-first-project.git
  cd Mobile-first-project
```

### Install

Install project dependencies in this project with:

```sh
  npm install
```

### Usage

To run the project, open the project with live server and visit [http://localhost:5500](http://localhost:5500)

### Open in browser

http://localhost:5500